package.json

```diff
@@ -70,7 +70,7 @@
     "pre-push": "yarn types:generate && yarn lint && yarn stylelint",
     "browser-test": "testcafe \"chrome --window-size='1920,1080'\" browser-tests/",
     "browser-test:local": "TEST_ENV=local testcafe \"chrome --window-size='1920,1080'\" browser-tests/",
```

Comment:
After these changes these seem to work identically. There is no longer a way to run browser tests against the staging environment, but is there a valid use case for that anyway?

Reply:
Yeah, couldn't come up with any valid use case for running the tests against the staging environment.